ext/liquid_c/expression.c

```diff
@@ -0,0 +1,100 @@
+#include "liquid.h"
+#include "vm_assembler.h"
+#include "parser.h"
+#include "vm.h"
+
+VALUE cLiquidCExpression;
+
+typedef struct expression {
+    vm_assembler_t code;
+} expression_t;
+
```

Comment:
If we include the header, this struct should be defined.

```suggestion
```

Reply:
Oops, I didn't mean to double include the header.  I'll fix that.